### ISSUE TITLE
solve problem using Downgrading Tensorflow 2.2.0rcl and keras 

### ISSUE THREAD

**Issue by panditrahulsharma** [keras-team/keras-applications#182]
i also faced lots of problem in Mask-RCNN but three codes that solve my problem
1.!pip uninstall tensorflow_estimator
2.!pip install tensorflow==1.13.2
3.!pip install 'keras==2.1.6' --force-reinstall
4.!python3 setup.py install
